base/io.jl

```diff
@@ -645,3 +645,34 @@ ismarked(io::IO) = io.mark >= 0
 Commit all currently buffered writes to the given stream.
 """
 flush(io::IO) = nothing
+
+"""
+    skipchars(io::IO, predicate; linecomment=nothing)
```

Comment:
there's a duplicate of this docstring now in `base/docs/helpdb/Base.jl` that should be removed